conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinStubs.scala

```diff
@@ -16,7 +16,7 @@ import tech.cryptonomic.conseil.common.rpc.RpcClient
   */
 trait BitcoinStubs {
 
-  implicit val contextShift: ContextShift[IO]
+  // implicit val contextShift: ContextShift[IO]
```

Comment:
why is this commented?

Reply:
`ContextShift` is not a thing in `cats effect` 3.0 anymore (https://typelevel.org/cats-effect/docs/migration-guide#contextshift)

Comment removed